packages/core/src/modules/generic-record/GenericRecordModule.ts

```diff
@@ -0,0 +1,77 @@
+import type { Logger } from '../../logger'
+import type { GenericRecord, GenericRecordTags, SaveGenericRecordOption } from './repository/GenericRecord'
+
+import { Lifecycle, scoped } from 'tsyringe'
+
+import { AgentConfig } from '../../agent/AgentConfig'
+
+import { GenericRecordService } from './service/GenericRecordService'
+
+export type ContentType = {
+  content: string
+}
+
+@scoped(Lifecycle.ContainerScoped)
+export class GenericRecordModule {
+  private genericRecordsService: GenericRecordService
+  private logger: Logger
+  public constructor(agentConfig: AgentConfig, genericRecordsService: GenericRecordService) {
+    this.genericRecordsService = genericRecordsService
+    this.logger = agentConfig.logger
+  }
+
+  public async save({ message, tags }: SaveGenericRecordOption) {
```

Comment:
I see it's called content in the service, would be good to keep it consistent

Reply:
Done